react-front-end/tsrc/search/SearchPageHelper.ts

```diff
@@ -411,3 +418,36 @@ export const writeRawModeToStorage = (value: boolean): void =>
 
 export const deleteRawModeFromStorage = (): void =>
   clearDataFromLocalStorage(RAW_MODE_STORAGE_KEY);
+
+/**
+ * This function returns an object which consists of a URL of Item Summary page and a onClick handler
+ * which is used to open the Summary page.
+ *
+ * @param uuid Item's UUID.
+ * @param version Item's version.
+ * @param history The History object used in the context, which is typically provided by calling 'useHistory' in components.
+ */
+export const builderOpenSummaryPageHandler = (
```

Comment:
Although it's a 'builder' function, our convention is just to prefix with `build`.

```suggestion
export const buildOpenSummaryPageHandler = (
```

Reply:
This must be because I have recently fought against builders :smile: